todo_app/app.py

```diff
@@ -2,19 +2,43 @@
 from todo_app.data.session_items import add_item, get_items
 
 from todo_app.flask_config import Config
+import requests
+import os
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    items = get_items()
+
+    url = "https://api.trello.com/1/boards/AbpSrXDQ/lists"
+
+    os.getenv("TRELLO_API_KEY")
+
+    querystring = {
+    "key":os.getenv("TRELLO_API_KEY"),
+    "token":os.getenv("TRELLO_API_TOKEN"),
+    "cards": "open" }
+
+    response = requests.request("GET", url, params=querystring)
+
+    response_json = response.json()
+
+    items = []
+
+    for trello_list in response_json:
+        for card in trello_list['cards']:
+            card['status'] = trello_list['name']
+            items.append(card)
+
     return render_template('index.html', items = items)
 
 @app.route('/create-a-todo', methods=['POST'])
 def create_new_todo():
     new_todo_title = request.form['todo-name']
     add_item(new_todo_title)
     return index()
+    
+
     
```

Comment:
You have missed out the part of the exercise where it asks you to let users mark items as "Done". All your items are forever "to do".

You wrote a function for talking to Trello but aren't using it - add a form to your HTML and a corresponding route to app.py that uses your `mark_card_done` function.